### PR TITLE
Slicewise voxel value rescaling within the task chunk

### PR DESCRIPTION
**Pull request by shangmu**
operator "normalize-section-mu"

### CODE REVIEW

File: chunkflow/flow.py [hunk 1]
```diff
@@ -276,6 +276,30 @@ def normalize_contrast_contrast_cmd(tasks, name, levels_path, mip, clip_fraction
             task['log']['timer'][name] = time() - start
         yield task
 
+@cli.command('normalize-section-mu')
```
Comment (jingpengw) [inline chunkflow/flow.py]: It would be better that the command name be the same with the operator name: `normalize-section-shang`

**Comment by jingpengw** [comment]
@shangmu I have changed your command name to be consistent with operator name. are you going to add unit test.

File: chunkflow/flow.py (hunk 1 above)
Comment (shangmu) [reply chunkflow/flow.py]: This was a pun intended as "mu as in μm/μs/etc" (micro, minimal, doesn't require a pre-computed histogram)

**Comment by shangmu** [comment]
I have unit test for my underlying function. Where do you have unit tests for the operators?

**Comment by jingpengw** [comment]
in the same place of operators

https://github.com/seung-lab/chunkflow/blob/master/chunkflow/operators/test_cutout.py
Jingpeng Wu
Postdoc in Princeton Neuroscience Institute, Princeton University, NJ
website: jingpengw.com

**Comment by jingpengw** [comment]
merge for now, you can add unit test later. @shangmu

**Pull request merged**